src/Shell/MigrationsShell.php

```diff
@@ -74,6 +74,7 @@ public function getOptionParser()
             ->addOption('template', ['short' => 't'])
             ->addOption('format', ['short' => 'f'])
             ->addOption('only', ['short' => 'o'])
```

Comment:
was d already taken?

Reply:
It's to keep compatible with Phinx, which has it at x.

Reply:
Ah, I see:

    -d, --date=DATE                    The date to migrate to

so indeed, we can use x here too now.